.travis.yml

```diff
@@ -2,8 +2,21 @@ language: generic
 # Use Ubuntu 16.04
 dist: xenial
 
+git:
+  clone: false  # Clone manually to work around Travis issues like https://github.com/travis-ci/travis-ci/issues/6337
+  depth: false  # Shallow clones can prevent diff against base branch
+  quiet: true
```

Comment:
(`clone: false` renders the other lines moot, but I kept them both for documentation and so they have the proper behavior in case someone changes `clone: true` later.)